vscode/src/commands.ts

```diff
@@ -23,7 +23,32 @@ const commandsMap: (state: ExtensionState) => {
       }
 
       vscode.window.showInformationMessage("Starting analyzer...");
-      analyzerClient.start();
+      if (fullScreenPanel && fullScreenPanel.webview) {
+        analyzerClient.start(fullScreenPanel.webview);
+      } else {
+        analyzerClient.start(state.sidebarProvider.webview!);
+      }
```

Comment:
wonder if we should make a helper that'll return the webview to make this easier